packages/component-library/styles/__test/tester.jsx

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import "./tester.scss"
```

Comment:
Added these files to a `__test` folder in `/styles` to prevent them from being mistaken as some kind of test for the entire package.